### ISSUE TITLE
Obsolete polytene chromosome, telomeric region ? 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#19177]
This probably represents an assay for GO:1990707 nuclear subtelomeric heterochromatin
@hattrill to check if this has impacts at FlyBase. 

Annotations are here: https://docs.google.com/spreadsheets/d/13lQS-jRawiwpaaJ50OCq-qTVTyQP1EB0xZ6kkfEHEQI/edit#gid=0

12 annotations or so 

Thanks, Pascale

**Comment by pgaudet** [comment]
@hattrill Did you have a chance to look at that ?

**Comment by hattrill** [comment]
Not yet....afraid seem to be attracting new tasks at the moment! (I'll make space for it at the end of the week)

**Comment by hattrill** [comment]
Hi @pgaudet I am fine with obsoleting polytene chromosome, telomeric region, but I am not so sure on using this nuclear subtelomeric heterochromatin as a replacement term. I will try to catchup on the telomere discussion ticket, but my feeling is that we just need to disguish between a telomerase/D loop type telomere and a retrotransposon repeat-type, as there are functional similarities. Need to get to grips with #2883 - must have missed some portion of this as some point :)

**Comment by ValWood** [comment]
So in summary, the new arrangement is

GO:0000781   chromosome, telomeric region
with these descendants

--GO:0140445    chromosome, telomeric repeat region (for the tandem repeats, D loop etc)

--GO:0099115    chromosome, subtelomeric region (for the non-tandem repeat part, you could use this for the retrotransposon repeats +SO)
----GO:1990707    nuclear subtelomeric heterochromatin 

GO:0000781   chromosome, telomeric region 
covers everything...

**Comment by hattrill** [comment]
Thanks @ValWood - that makes it easier - I got lost in the thread!

**Comment by hattrill** [comment]
@pgaudet I've moved the annotations polytene chromosome, telomeric region....so go ahead & obsolete.

Looking at the other polytene chromosome terms, I need to co-ordinate with the pheno/expression curators at FB to make sure we don't lose annotations.

**Comment by pgaudet** [comment]
Thanks @hattrill 

I have obsoleted 'polytene chromosome, telomeric region' - this is what this ticket is about. 

I'm leaving this ticket open if you want to review the other terms:

(175) polytene chromosome
(61)	polytene chromosome puff
(51)	polytene chromosome interband
(23)	polytene chromosome chromocenter
(20)	polytene chromosome band
(15)	polytene chromosome, telomeric region
(1)	alpha-heterochromatin
(1)	beta-heterochromatin


Thanks, Pascale

**Comment by hattrill** [comment]
Yes.....but on the back burner for a while. Too many things jumping the queue. Need to discuss with a few other FB curators first.

**Comment by pgaudet** [comment]
@hattrill Any update on this?

**Comment by hattrill** [comment]
No. Can we keep this open for a while - I need to do a clean up in FB and it's been used as a CV term in places other than GO that I need to worry about.

**Comment by pgaudet** [comment]
Sure no problem

**Comment by pgaudet** [comment]
Hi @hattrill 
Trying to clean up the transcription project - do you think you can have a look at that one soon?

**Comment by hattrill** [comment]
It's on my to do list, so I will get to it very soon

**Comment by hattrill** [comment]
Hi @pgaudet have consulted with people at FlyBase and am trying to find a way forward with the polytene stuff - some terms we will have to keep but some we can either merge or move.
